### DISCUSSION TITLE
jsdelivr源，消息中心>我的消息加载失败

### DISCUSSION THREAD

**Discussion by Vardariotai** [the1812/Bilibili-Evolved#3714] [综合讨论 / General]
chrome版本96，回复/点赞/通知均正常显示，但私信加载失败。
[message.bilibili.com-1664645900230.log](https://github.com/the1812/Bilibili-Evolved/files/9690869/message.bilibili.com-1664645900230.log)

**Comment by the1812** [comment]
请到 issues 里反馈
